logs/logWrapper.go

```diff
@@ -16,16 +20,22 @@ type stdbothWriter struct {
 }
 
 func (w stdoutWriter) Write(message []byte) (n int, err error) {
-	n, err = w.writer.Write(append([]byte("01 "), message...))
-	return n - 3, err
+	msg := append([]byte("01 "), message...)
```

Comment:
Nit: looks like we can extract repeatable code (23-26, 30-33, 37-40) to its own method?